src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+
+<figure class="w-figure">
+  <img src="./image8.png" alt="The Safari browser's integrated tile bar and toolbar.">
+  <figcaption class="w-figcaption"></figcaption>
+</figure>
+
+<figure class="w-figure">
+  <img src="./image1.png" alt="The Podcasts app's customized split customized title bar.">
+  <figcaption class="w-figcaption">Customized title bars of Safari and Podcasts.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  While not currently possible, <a href="https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/TitleBarCustomization/explainer.md">title bar customization</a> is being worked on at the moment.
+  You can (and should), however, specify the <a href="https://web.dev/add-manifest/#display"><code>display</code></a> and the
+  <a href="https://web.dev/add-manifest/#theme-color"><code>theme-color</code></a> properties of the Web App Manifest to
+  determine the look and feel of your application window and to decide which default browser controls—potentially none of them—should be shown.
+{% endDetails %}
+
+## Snappy animations
+
+In-app animations are snappy and smooth in Podcasts. For example, when I open the "Episode Notes" drawer on the right, it elegantly slides in. When I remove one episode from my downloads, the remaining episodes float up and consume the screen real estate that was freed by the deleted episode.
+
+<figure class="w-figure">
+  <img src="./image14.png" alt="The Podcasts app with the 'Episode Notes' drawer expanded." width="600">
+  <figcaption class="w-figcaption">In-app animations like when opening a drawer are snappy.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  Performant animations on the web are certainly possible if you take into account a number of best practices
+  outlined in the article <a href="https://developers.google.com/web/fundamentals/design-and-ux/animations/animations-and-performance">Animations and Performance</a>.
+  Scroll animations as commonly seen in paginated content or media carousels can be massively improved by using the <a href="https://developers.google.com/web/updates/2018/07/css-scroll-snap">CSS Scroll Snap</a> feature.
+  For full control, you can use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API">Web Animations API</a>.
+{% endDetails %}
+
+## Content surfaced outside of app
+
+The Podcasts app on iOS can surface content in other locations than the actual application, for example, in the system's Widgets view, or in the form of a Siri Suggestion. Having proactive, usage-based calls-to-action that just require a tap to interact with can greatly increase the re-engagement rate of an app like Podcasts.
+
+<figure class="w-figure">
+  <img src="./image6.png" alt="iOS Widget view showing the Podcasts app suggesting a new episode of a podcast." width="300">
```

Comment:
```suggestion
  <img src="./image6.png" class="w-screenshot w-screenshot--filled" alt="iOS Widget view showing the Podcasts app suggesting a new episode of a podcast." width="300">
```

Reply:
This image is quite busy. I suggest setting a plain black background and removing the **TIPS** and **STOCKS** notifications.

Reply:
I want it to be realistic, so not just one widget. I have added a red highlight box now.